packages/webapp/src/components/Table/styles.module.scss

```diff
@@ -69,6 +69,16 @@
     border-top: solid 1px transparent;
     border-bottom: solid 1px var(--grey200);
   }
+
+  &.removedRow {
+    background: var(--Colors-Neutral-Neutral-50);
+
+    & * {
+      color: var(--Colors-Neutral-Neutral-300);
+      font-style: italic;
+      padding-right: 2px; // compensate for italic glyphs being cut off by overflow: hidden
```

Comment:
I might have misunderstood the adjustments, but this makes the icon 2px smaller, which seems unintended.
Also, removed animal icons seem to be shifted 4px to the right...?

Reply:
Oooof yeah, that was a bad shortcut; I'll apply it to the proper component which is the text!

I wish there was a way around having to use padding to fix the the italic text being cut off though, it's really not the right tool...

Reply:
Okay, I moved the padding to the text and I _think_ (I've been looking at this component too long now to tell for sure) that this is a better solution for the lining up of icons and text; the animal icons now look centred to me relative to the batch ones now and the size should be the same for the removed ones as well (although I couldn't tell if a border or outline on the animal one would match better!)

@SayakaOno could you take another look at the icons? 🙏